airbyte-webapp/src/components/connectorBuilder/BuilderPrompt.tsx

```diff
@@ -0,0 +1,66 @@
+import { FormattedMessage } from "react-intl";
+import { useResizeDetector } from "react-resize-detector";
+import { useNavigate } from "react-router-dom";
+
+import { Button } from "components/ui/Button";
+import { FlexContainer } from "components/ui/Flex";
+import { Heading } from "components/ui/Heading";
+import { Text } from "components/ui/Text";
+
+import { ReactComponent as BuilderPromptIcon } from "./builder-prompt-icon.svg";
+import styles from "./BuilderPrompt.module.scss";
+
+interface BuilderPromptProps {
+  builderRoutePath: string;
+}
+
+export const BuilderPrompt: React.FC<BuilderPromptProps> = ({ builderRoutePath }) => {
+  const navigate = useNavigate();
+  const { width, ref } = useResizeDetector();
```

Comment:
The react-resize-detector library is used here to detect the width of the BuilderPrompt container, so that the `Build your connector` button can be moved underneath the text when the width gets small enough, as shown in this design: https://www.figma.com/file/F1ljvrJerLoVbNvtejBWid/LOW-CODE-CONNECTOR-BUILDER?node-id=3404%3A119487&t=EfcNzk8OWdDxBKoG-0